ee/clickhouse/queries/funnels/base.py

```diff
@@ -157,7 +161,14 @@ def _format_single_funnel(self, results, with_breakdown=False):
 
             if with_breakdown:
                 breakdown = results[-1]
-                serialized_result.update({"breakdown": breakdown, "breakdown_value": breakdown})
+                serialized_result.update(
+                    {
+                        "breakdown": get_breakdown_cohort_name(breakdown)
```

Comment:
non-blocking

can we wrap this in `()` so the formatting is a little less nuts?